vault/init.go

```diff
@@ -139,6 +143,37 @@ func (c *Core) Initialize(ctx context.Context, initParams *InitParams) (*InitRes
 		return nil, ErrAlreadyInit
 	}
 
+	// If we have clustered storage, set it up now
+	if raftStorage, ok := c.underlyingPhysical.(*raft.RaftBackend); ok {
+		// TODO: do we need to start cluster listener here? I think yes so we
```

Comment:
I wonder if we should mandate that `cluster_addr` be set if they're using Raft storage. It seems like a reasonable thing to do and then there aren't any questions about what port we might actually start the listener on. That's separate from whether or not the cluster listener should be started here, but if it's _only_ for that reason then probably it means we don't need to.